config/samples/xgboost-dist/xgboostjob_v1alpha1_iris_predict_local.yaml

```diff
@@ -42,7 +42,7 @@ spec:
               claimName: xgboostlocal
           containers:
           - name: xgboostjob
-            image: docker.io/merlintang/xgboost-dist-iris:1.1
+            image: docker.io/xfate123/xgboost-dist-iris:1.1
```

Comment:
Do we have a Docker file for this image in this repo?

Reply:
We only have image in this repo